### ISSUE TITLE
libflatbuffers.so.2：cannot open shared object file

### ISSUE THREAD

**Issue by tangyunlong** [conda-forge/tensorflow-feedstock#277]
### Comment:

I think the library file libflatbuffers.so.2 is missing.
Downloading the library file and linking it manually should solve the problem, but I can't find a download resource for this library
BTW, my running environment is as follows
```
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
absl-py                   1.2.0              pyhd8ed1ab_0    conda-forge
aiohttp                   3.8.1           py310h7f8727e_1  
aiosignal                 1.2.0              pyhd3eb1b0_0  
astunparse                1.6.3                      py_0  
async-timeout             4.0.2           py310h06a4308_0  
attrs                     21.4.0             pyhd3eb1b0_0  
blas                      1.0                         mkl  
blinker                   1.4             py310h06a4308_0  
brotlipy                  0.7.0           py310h7f8727e_1002  
bzip2                     1.0.8                h7b6447c_0  
c-ares                    1.18.1               h7f8727e_0  
ca-certificates           2022.07.19           h06a4308_0  
cachetools                4.2.2              pyhd3eb1b0_0  
certifi                   2022.9.24       py310h06a4308_0  
cffi                      1.15.1          py310h74dc2b5_0  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
click                     8.0.4           py310h06a4308_0  
cryptography              37.0.1          py310h9ce1e76_0  
cudatoolkit               11.3.1               h2bc3f7f_2  
cudnn                     8.4.1.50             hed8a83a_0    conda-forge
dataclasses               0.8                pyh6d0b6a4_7  
fftw                      3.3.9                h27cfd23_1  
flatbuffers               2.0.0                h2531618_0  
frozenlist                1.2.0           py310h7f8727e_1  
gast                      0.4.0              pyh9f0ad1d_0    conda-forge
giflib                    5.2.1                h7b6447c_0  
google-auth               2.6.0              pyhd3eb1b0_0  
google-auth-oauthlib      0.4.4              pyhd3eb1b0_0  
google-pasta              0.2.0              pyhd3eb1b0_0  
grpc-cpp                  1.47.1               h05bd8bd_6    conda-forge
grpcio                    1.47.1          py310h5b20944_6    conda-forge
h5py                      3.7.0           py310he06866b_0  
hdf5                      1.10.6               h3ffc7dd_1  
icu                       70.1                 h27087fc_0    conda-forge
idna                      3.3                pyhd3eb1b0_0  
importlib-metadata        4.11.3          py310h06a4308_0  
intel-openmp              2021.4.0          h06a4308_3561  
jpeg                      9e                   h7f8727e_0  
keras                     2.10.0             pyhd8ed1ab_0    conda-forge
keras-preprocessing       1.1.2              pyhd3eb1b0_0  
krb5                      1.19.2               hac12032_0  
ld_impl_linux-64          2.38                 h1181459_1  
libabseil                 20220623.0      cxx17_h48a1fff_4    conda-forge
libcurl                   7.84.0               h91b91d3_0  
libedit                   3.1.20210910         h7f8727e_0  
libev                     4.33                 h7f8727e_1  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 12.1.0              h8d9b700_16    conda-forge
libgfortran-ng            11.2.0               h00389a5_1  
libgfortran5              11.2.0               h1234567_1  
libgomp                   12.1.0              h8d9b700_16    conda-forge
libnghttp2                1.46.0               hce63b2e_0  
libpng                    1.6.37               hbc83047_0  
libprotobuf               3.21.7               h6239696_0    conda-forge
libsqlite                 3.39.4               h753d276_0    conda-forge
libssh2                   1.10.0               h8f2d780_0  
libstdcxx-ng              12.1.0              ha89aaad_16    conda-forge
libuuid                   1.0.3                h7f8727e_2  
libzlib                   1.2.12               h166bdaf_3    conda-forge
markdown                  3.3.4           py310h06a4308_0  
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0           py310h7f8727e_0  
mkl_fft                   1.3.1           py310hd6ae3a3_0  
mkl_random                1.2.2           py310h00e6091_0  
multidict                 5.2.0           py310h5eee18b_3  
nccl                      2.14.3.1             h0800d71_0    conda-forge
ncurses                   6.3                  h5eee18b_3  
numpy                     1.23.1          py310h1794996_0  
numpy-base                1.23.1          py310hcba007f_0  
oauthlib                  3.2.0              pyhd3eb1b0_1  
openssl                   1.1.1q               h7f8727e_0  
opt_einsum                3.3.0              pyhd3eb1b0_1  
packaging                 21.3               pyhd3eb1b0_0  
pip                       22.2.2          py310h06a4308_0  
protobuf                  4.21.7          py310hd8f1fbe_0    conda-forge
pyasn1                    0.4.8              pyhd3eb1b0_0  
pyasn1-modules            0.2.8                      py_0  
pycparser                 2.21               pyhd3eb1b0_0  
pyjwt                     2.4.0           py310h06a4308_0  
pyopenssl                 22.0.0             pyhd3eb1b0_0  
pyparsing                 3.0.9           py310h06a4308_0  
pysocks                   1.7.1           py310h06a4308_0  
python                    3.10.6               haa1d7c7_0  
python-flatbuffers        2.0                pyhd3eb1b0_0  
python_abi                3.10                    2_cp310    conda-forge
re2                       2022.06.01           h27087fc_0    conda-forge
readline                  8.1.2                h7f8727e_1  
requests                  2.28.1          py310h06a4308_0  
requests-oauthlib         1.3.0                      py_0  
rsa                       4.7.2              pyhd3eb1b0_1  
scipy                     1.9.1           py310hd5efca6_0  
setuptools                63.4.1          py310h06a4308_0  
six                       1.16.0             pyhd3eb1b0_1  
snappy                    1.1.9                h295c915_0  
sqlite                    3.39.3               h5082296_0  
tensorboard               2.10.1             pyhd8ed1ab_0    conda-forge
tensorboard-data-server   0.6.0           py310hca6d32c_0  
tensorboard-plugin-wit    1.8.1           py310h06a4308_0  
tensorflow                2.10.0          cuda112py310he87a039_0    conda-forge
tensorflow-base           2.10.0          cuda112py310hf679b68_0    conda-forge
tensorflow-estimator      2.10.0          cuda112py310h2fa73eb_0    conda-forge
tensorflow-gpu            2.10.0          cuda112py310h0bbbad9_0    conda-forge
termcolor                 1.1.0           py310h06a4308_1  
tk                        8.6.12               h1ccaba5_0  
typing_extensions         4.3.0           py310h06a4308_0  
tzdata                    2022c                h04d1e81_0  
urllib3                   1.26.11         py310h06a4308_0  
werkzeug                  2.0.3              pyhd3eb1b0_0  
wheel                     0.37.1             pyhd3eb1b0_0  
wrapt                     1.14.1          py310h5eee18b_0  
xz                        5.2.6                h5eee18b_0  
yarl                      1.8.1           py310h5eee18b_0  
zipp                      3.8.0           py310h06a4308_0  
zlib                      1.2.12               h5eee18b_3  
```

and 

GPU is NVIDIA GeForce 3090
NVIDIA-SMI 515.65.01    Driver Version: 515.65.01    CUDA Version: 11.7

**Comment by hmaarrfk** [comment]
Let me know if you need any addition i formation that the one contained in the comment

https://github.com/conda-forge/tensorflow-feedstock/issues/275#issuecomment-1258408112

**Comment by tangyunlong** [comment]
ok thank you a lot

**Comment by remingm** [comment]
The fix from #275 didn't work for me, I was already strictly using conda-forge. I found the following files in my env:

```
/home/username/miniconda3/envs/env-name/lib/libflatbuffers.a
/home/username/miniconda3/envs/env-name/lib/libflatbuffers.so
/home/username/miniconda3/envs/env-name/lib/libflatbuffers.so.22
/home/username/miniconda3/envs/env-name/lib/libflatbuffers.so.22.12.06
```
I simply copied libflatbuffers.so.22 to the same location as libflatbuffers.so.2 and Tensorflow 2.10 stopped throwing the error.

**Comment by hmaarrfk** [comment]
@remingm sorry that #275 didn't work for you.

I would be glad to help troubleshoot, but the information required in the new issue template is critical.

I'm going to close this issue since I seem to have resolved @tangyunlong .

If you are indeed experience the issue, strictly with conda-forge, then the issue is somewhere else. Maybe a real bug in our system. Please do open an issue. You should not have to copy `so` files around. conda should do that for you!
